app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs

```diff
@@ -662,6 +662,19 @@
                                   (fn [e!]
                                     (e! (close-event))
                                     (e! (cooperation-controller/->ResponseCreated response (boolean (:db/id @form-atom)))))))
+                 :delete (when (:db/id @form-atom)
```

Comment:
Does this need some authorization checking? Or should the defcommands pre check be used here to show a disabled delete button when an opinion exists?

Reply:
If you can create/edit, then you can also delete afaict

Reply:
The pre check is already checked, the "edit" button isn't shown if the application has an opinion